src/openzaak/components/catalogi/tests/test_zaaktype.py

```diff
@@ -1421,6 +1430,136 @@ def test_update_published_zaaktype_requires_forced_write_scope(self):
+            "vertrouwelijkheidaanduiding": VertrouwelijkheidsAanduiding.openbaar,
+            "omschrijving": "PUT bron test",
+            "gerelateerdeZaaktypen": [],
+            "referentieproces": {"naam": "ReferentieProces 0", "link": ""},
+            "catalogus": reverse(catalog2),
+            "besluittypen": [],
+            "beginGeldigheid": "2018-01-01",
+            "versiedatum": "2018-01-01",
+            "verantwoordelijke": "063308836",
+            "broncatalogus": {
+                "url": f"http://example.com{reverse(catalog1)}",
+                "domein": catalog1.domein,
+                "rsin": catalog1.rsin,
+            },
+            "bronzaaktype": {
+                "url": f"http://example.com{reverse(bron_zaaktype)}",
+                "identificatie": bron_zaaktype.identificatie,
+                "omschrijving": bron_zaaktype.zaaktype_omschrijving,
+            },
+        }
+
+        response = self.client.put(zaaktype_url, data, format="json")
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
+
+        zaaktype.refresh_from_db()
+        self.assertEqual(zaaktype.broncatalogus_domein, catalog1.domein)
+        self.assertEqual(zaaktype.broncatalogus_rsin, catalog1.rsin)
+        self.assertEqual(
+            zaaktype.bronzaaktype_identificatie, bron_zaaktype.identificatie
+        )
+        self.assertEqual(
+            zaaktype.bronzaaktype_omschrijving, bron_zaaktype.zaaktype_omschrijving
+        )
+
+    def test_patch_update_zaaktype_with_bron_fields(self):
+        catalog1 = CatalogusFactory.create(domein="PATCH", rsin="517439943")
+        catalog2 = CatalogusFactory.create(domein="OTHER", rsin="004455667")
+
+        bron_zaaktype = ZaakTypeFactory.create(catalogus=catalog1)
+
+        zaaktype = ZaakTypeFactory.create(catalogus=catalog2)
+        zaaktype_url = reverse(zaaktype)
+
+        patch_data = {
+            "broncatalogus": {
+                "domein": catalog1.domein,
+                "rsin": catalog1.rsin,
+                "url": f"http://example.com{reverse(catalog1)}",
+            },
+            "bronzaaktype": {
+                "identificatie": bron_zaaktype.identificatie,
+                "omschrijving": bron_zaaktype.zaaktype_omschrijving,
+                "url": f"http://example.com{reverse(bron_zaaktype)}",
+            },
+        }
+
+        response = self.client.patch(zaaktype_url, patch_data, format="json")
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
+
+        zaaktype.refresh_from_db()
+        self.assertEqual(zaaktype.broncatalogus_domein, catalog1.domein)
+        self.assertEqual(zaaktype.broncatalogus_rsin, catalog1.rsin)
+        self.assertEqual(
+            zaaktype.bronzaaktype_identificatie, bron_zaaktype.identificatie
+        )
+        self.assertEqual(
+            zaaktype.bronzaaktype_omschrijving, bron_zaaktype.zaaktype_omschrijving
+        )
```

Comment:
Could you also add the assertions for `broncatalogus_url` and `bronzaaktype_url` here?